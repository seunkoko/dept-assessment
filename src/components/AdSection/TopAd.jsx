// eslint-disable-next-line
import React from 'react';

import ImageContainer from '../ImageContainer/ImageContainer';
import AdContent from './AdContent';

import './Ad.scss';

import { imageData, topFirstNews, topSecondNews } from '../../utils/adData';

const TopAd = (props) => {
	return (
    <>
      <div className="top-ad-style">
        <div className="image-ad">
          <ImageContainer
            data={
              {
                ...imageData,
                id: 'topAdImage',
              }
            }
          />
        </div>

        <div className="news-content">
          <AdContent
            firstNews={topFirstNews}
            secondNews={topSecondNews}
          />
        </div>
      </div>
    </>
	);
};

export default TopAd;
