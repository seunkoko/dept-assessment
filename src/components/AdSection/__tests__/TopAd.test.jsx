import React from 'react';

import { shallow } from 'enzyme';

import TopAd from '../TopAd';

describe('TopAd', () => {
	const component = shallow(<TopAd />);

	it('should render TopAd component', () => {
		expect(component).toMatchSnapshot();
  });
  
});