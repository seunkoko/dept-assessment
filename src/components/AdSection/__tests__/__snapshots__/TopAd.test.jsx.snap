// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TopAd should render TopAd component 1`] = `
<Fragment>
  <div
    className="top-ad-style"
  >
    <div
      className="image-ad"
    >
      <ImageContainer
        data={
          Object {
            "description": "Rethinking the online ecosystem",
            "id": "topAdImage",
            "image": "test-file-stub",
            "moreDescription": "Anim pariatur cliche reprehenderit, enim eiusmod high life accusamus terry richardson ad squid. Nihil anim keffiyeh helvetica, craft beer labore wes anderson cred nesciunt sapiente ea proident.",
            "title": "Florensis",
          }
        }
      />
    </div>
    <div
      className="news-content"
    >
      <AdContent
        firstNews={
          Object {
            "id": "topFirstNews",
            "moreDescription": "",
            "news": "Tapping into Ireland’s unique gaming culture and bringing a fresh flavour to their Xbox social media channels",
            "title": "MICROSOFT",
          }
        }
        secondNews={
          Object {
            "id": "topSecondNews",
            "moreDescription": "",
            "news": "Integrating existing content into O’Neill’s new e-commerce platform",
            "title": "O'NEIL",
          }
        }
      />
    </div>
  </div>
</Fragment>
`;
